### PR TITLE
Pocket Casts for Linux

### PR DESCRIPTION
**Pull request by hadess**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://github.com/felicianotech/pocket-casts-desktop-app/issues/42
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: tech.feliciano.pocket-casts.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```
Comment (barthalion) [inline tech.feliciano.pocket-casts.yml:13]:
```suggestion
  - --socket=x11
```

**Comment by barthalion** [comment]
bot, build tech.feliciano.pocket-casts

**Comment by barthalion** [comment]
Appdata validation errors:

```
builddir/files/share/appdata/tech.feliciano.pocket-casts.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
```

aarch64 failed:

```
$ electron-builder --dir
  • electron-builder  version=22.7.0 os=5.4.0-62-generic
  • loaded configuration  file=package.json ("build" field)
  • writing effective config  file=dist/builder-effective-config.yaml
  • packaging       platform=linux arch=arm64 electron=9.3.1 appOutDir=dist/linux-arm64-unpacked
```

vs

```
cp: cannot stat 'dist/linux-unpacked/': No such file or directory
```

File: tech.feliciano.pocket-casts.yml (hunk 1 above)
Comment (hadess) [reply tech.feliciano.pocket-casts.yml:13]: No Wayland support there?

**Comment by hadess** [comment]
Should be all fixed now.

**Comment by hadess** [comment]
bot, build tech.feliciano.pocket-casts

File: tech.feliciano.pocket-casts.yml (hunk 1 above)
Comment (nedrichards) [reply tech.feliciano.pocket-casts.yml]: Not in the chromium-verse yet (without flags twiddling and special compiles). Should be in mainline chromium soon, thence to electron eventually.

File: tech.feliciano.pocket-casts.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  - org.freedesktop.Sdk.Extension.node14
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
```
Comment (nedrichards) [inline tech.feliciano.pocket-casts.yml:15]:
```suggestion
  - --share=network
  - --device=dri
```

will address

> libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: i965

**Comment by hadess** [comment]
bot, build tech.feliciano.pocket-casts

**Comment by nedrichards** [comment]
This lgtm and I'm keen to use it myself. I'll give a few days for upstream to respond to the issue linked if they wish.

**Comment by nedrichards** [comment]
/merge

One week is a good amount of time to wait. As ever it's very easy to add upstream contributors after the fact if required.

**Pull request closed without merging**